site/content/docs/5.3/migration.md

```diff
@@ -23,10 +23,13 @@ toc: true
 #### Buttons
   - <span class="badge text-bg-success">New</span> X social button.
 
+#### Orange navbar
```

Comment:
I'm going crazy with all these PRs... This was already done in #2690 [here](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2690/files#diff-ff554e03e75f214021058bea325f4efdbe435154b706ecf220c9b29653a4ba8b), and in a better way I find... What do you think @julien-deramond

Reply:
Anyway, I made some changes: 
- fix only `role="img"` in this PR #2704 => Change migration guide accordingly
- fix only alt attribute in PR #2690 => Change migration guide accordingly